pages/events/index.vue

```diff
@@ -52,31 +40,44 @@
               />
             </client-only>
           </AppFieldset>
-        </div>
-        <div class="bx--col-lg-12 bx--col-md-6 event-page__main-content">
-          <div>
-            <AppCard
-              v-if="noEvents"
-              :image="emptyCard.img"
-              :title="emptyCard.title"
-            >
-              <div class="event-page__empty-card-description">
-                {{ emptyCard.description }}
-              </div>
-            </AppCard>
-            <EventCard
-              v-for="event in filteredEvents"
-              v-else
-              :key="`${event.title}-${event.place}-${event.date}`"
-              class="event-page__event-card"
-              :types="event.types"
-              :title="event.title"
-              :image="event.image"
-              :location="event.location"
-              :date="event.date"
-              :to="event.to"
+        </template>
+        <template slot="filters-on-s-screen">
+          <div
+            v-for="filter in extraFilters"
+            :key="filter.label"
+          >
+            <AppMultiSelect
+              :label="filter.label"
+              :options="filter.options"
+              :value="getCheckedFilters(filter.filterType)"
+              @change-selection="updateWholeFilter(filter.filterType, $event)"
             />
           </div>
+        </template>
+        <template slot="results">
+          <AppCard
+            v-if="noEvents"
+            :image="emptyCard.img"
+            :title="emptyCard.title"
+          >
+            <div class="event-page__empty-card-description">
+              {{ emptyCard.description }}
+            </div>
+          </AppCard>
+          <EventCard
+            v-for="event in filteredEvents"
+            v-else
+            :key="`${event.title}-${event.place}-${event.date}`"
+            class="results-item"
```

Comment:
I find it quite tricky to figure out where this definition is without prior knowledge (like this PRs code review).

I'm think of another approach that could be easier to understand.

1. Maybe we could define `.results.item` in the component using the definition (**pages/events/index.vue** and **pages/learn.index.vue**).
2. Or maybe we could leave the definition in **components/ui/AppFiltersResultsLayout.vue** but rename it to `.app-filters-results-layout__results-item` to hint to where the definition is.

Reply:
I thought the same when I did it 👍 . I like the second option